localization_manager/test/node_test.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <gtest/gtest.h>
+#include <memory>
+#include <chrono>
+#include <thread>
+#include <future>
+
+#include "localization_manager/localization_manager_node.hpp"
+
+
+// TODO for USER: Implement a real test using GTest
+TEST(Testlocalization_manager, example_test){
+
+    rclcpp::NodeOptions options;
+    auto worker_node = std::make_shared<localization_manager::Node>(options);
+
+    worker_node->configure(); //Call configure state transition
+    worker_node->activate();  //Call activate state transition to get not read for runtime
+
+    std::unique_ptr<std_msgs::msg::String> msg = std::make_unique<std_msgs::msg::String>();
+    msg->data = "my string";
+
+    worker_node->example_callback(move(msg)); // Manually drive topic callbacks
+
+}
```

Comment:
This test should be removed. Also example_callback shouldn't compile right? Have the unit tests been run?

Reply:
yes you're right this doesn't compile, it wasn't included in the cmake. This file is not required and is removed now